Libplanet/Blocks/IBlockContent.cs

```diff
@@ -0,0 +1,24 @@
+#nullable enable
+using System.Collections.Generic;
+using Libplanet.Action;
+using Libplanet.Tx;
+
+namespace Libplanet.Blocks
+{
+    /// <summary>
+    /// A common interface for blocks that do not have any proofs, but have their metadata and
+    /// containing <see cref="Transactions"/>.
+    /// </summary>
+    /// <typeparam name="T">A class implementing <see cref="IAction"/> to include.  This type
+    /// parameter is aligned with <see cref="Transaction{T}"/>'s type parameter.</typeparam>
+    public interface IBlockContent<T> : IBlockMetadata
+        where T : IAction, new()
+    {
+        /// <summary>
+        /// Transactions belonging to the block.
+        /// </summary>
+        /// <remarks>This is always ordered by <see cref="Transaction{T}.Id"/>.</remarks>
+        // FIXME: Need to change the type to IImmutableSet<Transaction<T>>.
+        IReadOnlyList<Transaction<T>> Transactions { get; }
```

Comment:
```suggestion
        IImmutableList<Transaction<T>> Transactions { get; }
```

I think `IImmutableList<T>` might be more appropriate considering its underlying class. Depending on type compatibility, we can ignore this for now.

Reply:
As I left a `FIXME` comment, IMHO ideally it should be `IImmutableSet<Transaction<T>>` so that client code can't depend on its implicit ordering.  When a certain ordering is needed, it should explicitly sort them in client side.  However, changing the type seems out of the scope of this patch.  It would be good if we have a separated issue for discussing it.